indico/htdocs/js/indico/modules/timetable/timetable/Management.js

```diff
@@ -1068,3 +1068,84 @@ type("FitInnerTimetableDialog", ["ConfirmPopup"], {
         this.ConfirmPopup($T("Fit timetable to content"), this.__getContent(), this.__handler);
     }
 );
+
+
+type("SessionSectionPopupMenu", ["SectionPopupMenu"],
+    {
+        _processItem: function(pair) {
+            var self = this;
+            var value = pair.get();
+            var color = null;
+            var title = null;
+
+            if(exists(value.title)){
```

Comment:
Super-duper-ultra legacy. Simply moved it from a common JS to the timetable module. I wouldn't spend time on making this pretty.